Controller/H5PInteractionController.php

```diff
@@ -153,43 +174,13 @@ public function embedAction(Request $request, Content $content)
                 ],
             ],
         ];
-        $h5p_content = $content;
```

Comment:
I'm not sure whether this part of the code should be deleted, can you tell me why?

Reply:
Yes you are right, i deleted the wrong side of the `if`. Thanks for checking!